### DISCUSSION TITLE
Fazer upload somente dos arquivos selecionados

### DISCUSSION THREAD

**Discussion by Joel-Sousa** [Natizyskunk/vscode-sftp#351] [Ideas]
Tentei fazer upload de determindados arquivos selecionados mas sempre ele fazia upload da pasta que estava contido os arquivos e pastas contidos, seria interessante se ele fizesse somente upload do arquivo selecionado e colocasse no caminho apontado independentemente do diretorio que ele esteja

**Comment by Joel-Sousa** [comment]
Descobri que tem como ataves da propriedade contexto:

"context": "nome da pasta"

**Comment by Natizyskunk** [comment]
@Joel-Sousa I don't understand. Could you please write in English ? 🙂

**Reply by Natizyskunk** [reply]
@Joel-Sousa Could you also open an issue respecting [this](https://github.com/Natizyskunk/vscode-sftp/issues/new?assignees=&labels=&projects=&template=bug_report.md&title=) template please ?
